proposals/distributed-async-epoch-reward.md

```diff
@@ -133,235 +128,110 @@ struct RewardHistory {
+distribution mechanism, and to minimize the risk of unintended consequences or
+disruptions to the network's operations.
 
-         1. When taking snapshot reward_distribution, reward_calculation_result
-            will be serialized and stored in the snapshot.
+## Security Considerations
 
-      - Restarting:
+Epoch rewards in a blockchain system are an important component that helps to
+incentivize network participants to secure the network and maintain its
+operations. Changing epoch rewards can have a significant impact on the
+network's overall stability and the behavior of network participants.
+Therefore, any changes to epoch rewards should be carefully considered and
+tested before rolling out.
 
-         1. When restarting from a snapshot during reward_calculation, a reward
-            calculation request will be recreated from the snap of stake_cache
-            at E, deserialized from the snapshot and sent to
-            reward_calculation_service to restart the reward calculation.
+The addition of sysvar accounts can provide some additional safeguards to
+prevent large capital changes, but it is important to note that they do not
+eliminate all risks associated with changing epoch rewards.
 
-         1. When restarting from a snapshot during reward_distribution, the
-            reward calculation result will be loaded and deserialized from the
-            snapshot. The merkle tree reward proof from the reward result will
-            be recalculated. The recalculated reward calculation result and
-            reward proof are used to distribute rewards.
+## Backwards Compatibility
 
-
-1.  How to "gate" these redemption transactions to only be run by validators? If
-    the redemption happens in a transaction, then technically anyone with SOL
-    can send it to the network.
-
-    Require (only) transaction signer be the slot leader?
-
-1. Since stake accounts aren't locked, if someone deactivates their stake during
-   epoch 100, and it becomes inactive at the start of epoch 101, they can
-   withdraw all of the lamports right away. What happens to their rewards from
-   epoch 100? Will it become claimable by anyone who has the private key to the
-   account after it's cleaned up by the runtime?
-
-   Yes, anyone with the private key can withdraw the rewards. However, a lot of
-   the stake account keys are discarded when the account is closed. it is better
-   to prevent this from happening. [Updated in the main text]
-
-1. During reward distribution, the reward proof verification might fail, i.e.
-   the rewards result are corrupted. This can be caused by a bug or for some
-   unforeseeable reason. If this happens, the reward for that particular account
-   will be lost. However, we will be protected from blowing up the total capital
-   with excessive rewards. Is this behavior reasonable?
+The new epoch payout approach will not be incompatible with the old approach.
```

Comment:
```suggestion
The new epoch payout approach will now be incompatible with the old approach.
```

Reply:
Maybe it is still compatible? My reading of it seems it will not be, but regardless the wording of the sentence is confusing.

Reply:
Good catch. should be `not compatible`